src/usage_metrics/resources/sqlite.py

```diff
@@ -49,26 +50,43 @@ def append_df_to_table(self, df: pd.DataFrame, table_name: str) -> None:
             table_name in usage_metrics_metadata.tables
         ), f"""{table_name} does not have a database schema defined.
             Create a schema one in usage_metrics.models."""
+        table_obj = usage_metrics_metadata.tables[table_name]
 
         if self.clobber:
```

Comment:
Do we still need the clobber argument now that we are using alembic?

Reply:
Just went ahead and removed it.